smithy-aws-protocol-tests/model/rpcV2/cbor-maps.smithy

```diff
@@ -9,11 +9,13 @@ use aws.protocoltests.shared#StringSet
 use smithy.test#httpRequestTests
 use smithy.test#httpResponseTests
 use smithy.protocols#rpcv2Cbor
+use smithy.framework#ValidationException
 
 /// The example tests basic map serialization.
 operation RpcV2CborMaps {
     input: RpcV2CborMapsInputOutput,
-    output: RpcV2CborMapsInputOutput
+    output: RpcV2CborMapsInputOutput,
+    errors: [ValidationException]
```

Comment:
This is necessary because `RpcV2CborMapsInputOutput` contains a member `denseSetMap` of the target type `DenseSetMap`, which in turn contains a member of the target type `StringSet` with the `@uniqueItems` trait applied to it. Consequently, SmithyRs requires that `ValidationException` be included as one of the possible errors that can be returned.